packages/core/src/polls.ts

```diff
@@ -12,18 +12,12 @@ class Polls {
     interval: number,
     cb: VoidFunction,
     options: PollOptions,
-  ): {
-    stop: VoidFunction
-    start: VoidFunction
-  } {
+  ): Poll {
     const poll = new Poll(interval, cb, options)
 
     this.polls.push(poll)
 
-    return {
-      stop: () => poll.stop(),
-      start: () => poll.start(),
-    }
+    return poll
```

Comment:
I wasn't sure why the reference gets broken here, but it's needed to keep the reactivity later down the line